src/CodeStyle/Core/CodeFixes/Host/Mef/CodeStyleHostLanguageServices.MefHostExportProvider.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Composition;
+using System.Composition.Hosting;
+using System.Linq;
+using System.Reflection;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.CodeAnalysis.PooledObjects;
+
+namespace Microsoft.CodeAnalysis.Host
+{
+    internal sealed partial class CodeStyleHostLanguageServices : HostLanguageServices
+    {
+        private sealed class MefHostExportProvider : IMefHostExportProvider
+        {
+            private readonly CompositionHost _compositionContext;
+
+            public MefHostExportProvider(string languageName)
+            {
+                var assemblies = CreateAssemblies(languageName);
+                var compositionConfiguration = new ContainerConfiguration().WithAssemblies(assemblies);
+                _compositionContext = compositionConfiguration.CreateContainer();
```

Comment:
📝 The use of MEF still needs to be removed.

Reply:
Yes, this is tracked with a follow up item: https://github.com/dotnet/roslyn/issues/41462